runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java

```diff
@@ -163,4 +163,15 @@ default boolean showAgilityArenaTimer()
 	{
 		return true;
 	}
+
+	@ConfigItem(
```

Comment:
I would probably move this up to position 0 since it is the main purpose of the plugin